components/button/demo/loading.md

```diff
@@ -1,10 +1,18 @@
 ---
 order: 4
-title: 加载中
+title:
+  zh-CN: 加载中状态
+  en-US: Loading
 ---
 
+## zh-CN
+
 添加 `loading` 属性即可让按钮处于加载状态，最后两个按钮演示点击后进入加载状态。
 
+## en-US
+
+Set `loading` to `Button`, and then it will become loading button.
```

Comment:
A loading indicator can be added to a button by setting the `loading` attribute on the `Button`.